website/screens/components/box/code/examples/Basic.ts

```diff
@@ -0,0 +1,27 @@
+import { DxcBox, DxcInset, DxcStack } from "@dxc-technology/halstack-react";
+
+const code = `() => {
+    return (
+        <DxcInset space="large">
+          <DxcStack gutter="large">
+            <DxcBox shadowDepth={0} padding="medium">
+                Box Content
+            </DxcBox>
+            <DxcBox shadowDepth={1} padding="medium">
+                Box Content
+            </DxcBox>
+            <DxcBox shadowDepth={2} padding="medium">
+                Box Content
+            </DxcBox>
+          </DxcStack>
+        </DxcInset>
```

Comment:
Something strange is happening here, since the box default size is fitContent but its showing like fillParent.

Reply:
Yes, this is caused by the DxcStack.

Reply:
It can be fixed if instead  of `flex` it is `inline-flex`